.github/workflows/frontend-lint-test-build.yaml

```diff
@@ -0,0 +1,266 @@
+name: Frontend
+on:
+  pull_request
+
+jobs:
+  # In order to be able to have required checks, a workflow can't be entirely
+  # skipped: see https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+  paths-filter:
```

Comment:
Thanks for adding this info, was wondering why this was used instead of standard paths filter